pkg/loghttp/params.go

```diff
@@ -48,14 +48,40 @@ func shards(r *http.Request) []string {
 
 func bounds(r *http.Request) (time.Time, time.Time, error) {
 	now := time.Now()
-	start, err := parseTimestamp(r.Form.Get("start"), now.Add(-defaultSince))
+	start := r.Form.Get("start")
+	end := r.Form.Get("end")
+	since := r.Form.Get("since")
+	return determineBounds(now, start, end, since)
+}
+
+func determineBounds(now time.Time, startString, endString, sinceString string) (time.Time, time.Time, error) {
+	since := defaultSince
+	if sinceString != "" {
+		d, err := model.ParseDuration(sinceString)
+		if err != nil {
+			return time.Time{}, time.Time{}, errors.Wrap(err, "could not parse 'since' parameter")
+		}
+		since = time.Duration(d)
+	}
```

Comment:
double checking: `parseTimestamp(sinceString, defaultSince)` won't work here?

Reply:
Since is a _duration_, not a _time_ so parsing it as a time won't work :(